.storybook/main.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  stories: [],
+  addons: ['@storybook/addon-essentials', '@storybook/addon-links', '@storybook/preset-scss'],
```

Comment:
Consider moving these to lib specific config until they have proven to be universal configs.

Reply:
Fixed